### PR TITLE
Tool suggestions

### PR DESCRIPTION
**Pull request by iveksl2**
@peterhurford @robertzk

### CODE REVIEW

**Comment by iveksl2** [comment]
@peterhurford still a python newb so I haven't used any of these myself. I wrote down suggestions from  @adamtheturtle talk [reviewing code well](https://www.youtube.com/watch?v=JWBBI-jFVOc&feature=youtu.be) . Thought id throw them into this repo

**Comment by peterhurford** [comment]
Nice, I haven't heard of any of these except **flake8**, which I use. 👍

File: suggestions.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+   * [requires.io](https://requires.io/)
+
+Quality metrics:
+   * [landscape.io](https://landscape.io/)
+
+Code Reviews:
+   * [reviewable.io](https://reviewable.io/)
+
+Test Coverage:
+   * [coverage.py](https://www.pylint.org://coverage.readthedocs.io/en/coverage-4.3.1/)
```
Comment (peterhurford) [inline suggestions.md]: ^ This link is broken.

**Review by peterhurford — CHANGES_REQUESTED** [review]

File: suggestions.md (hunk 1 above)
Comment (peterhurford) [reply suggestions.md]: Also worth noting that Codecov and Coveralls both work for Python.

File: suggestions.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+**Tools:**
+
+Style Guide / Linters:
+  * [flake8](https://pypi.python.org/pypi/flake8)
+  * [pylint](https://www.pylint.org/)
+  * [doc8](https://pypi.python.org/pypi/doc8)
+
+Dependency Tracking:
+   * [requires.io](https://requires.io/)
```
Comment (peterhurford) [inline suggestions.md:9]: Looks cool!

File: suggestions.md (hunk 1 above)
Comment (iveksl2) [reply suggestions.md]: changed, thx!

**Pull request merged**